Sources/KlaviyoCore/AppContextInfo.swift

```diff
@@ -7,18 +7,18 @@
 import Foundation
 import UIKit
 
-struct AppContextInfo {
```

Comment:
Moved this file to core and exposing it's members publicly so that `KlaviyoSwift` can access them. This is the case of all the files that were moved to `KlaviyoCore` and need to be accessed from `KlaviyoSwift`